templates/wheretofindme/follower_list.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block "main" %}
+<h1 class="display-4">Who follows me?</h1>
+<div>
+  {% for follow in object_list %}
+    <div class="card bg-light shadow-sm mb-3">
+      <a href="{% url "user-profile" slug=follow.from_user.username %}">
```

Comment:
The same point about showing a pivot table I made on #29 applies here. Being able to quickly scan the list and see _who these people are_ is probably useful.